internal/command/user_idp_link.go

```diff
@@ -15,15 +15,22 @@ func (c *Commands) AddUserIDPLink(ctx context.Context, userID, resourceOwner str
 	if userID == "" {
 		return nil, zerrors.ThrowInvalidArgument(nil, "COMMAND-03j8f", "Errors.IDMissing")
 	}
-	if err := c.checkUserExists(ctx, userID, resourceOwner); err != nil {
+
```

Comment:
among others, the function is called from the user v2 service and that still passes the org id from context

Reply:
and i noticed the `authz.GetCtxData(ctx).OrgID` is also still used elsewhere in that service, is that intended?

Reply:
No, that's exactly the spot where I wanted to remove it from, is removed.